interface/grpc/core/deploy_integration_test.go

```diff
@@ -17,9 +17,10 @@ func TestIntegrationDeployService(t *testing.T) {
 	stream := newTestDeployStream(url)
 
 	require.Nil(t, server.DeployService(stream))
-	defer server.api.DeleteService(stream.serviceID, false)
+	defer server.api.DeleteService(stream.sid, false)
```

Comment:
```suggestion
	defer server.api.DeleteService(stream.hash, false)
```
let's keep hash everywhere